Sources/_StringProcessing/MatchingOptions.swift

```diff
@@ -274,7 +292,15 @@ extension MatchingOptions.Representation {
   static var semanticMatchingLevels: Self {
     [.graphemeClusterSemantics, .unicodeScalarSemantics, .byteSemantics]
   }
-    
+  
+  // Quantification behavior options
+  static var reluctantByDefault: Self { .init(.reluctantByDefault) }
+  static var possessiveByDefault: Self { .init(.possessiveByDefault) }
+
+  static var quantificationBehaviors: Self {
+    [.reluctantByDefault, .possessiveByDefault]
```

Comment:
Couldn't someone switch the default back to eager via nesting?

Reply:
Yes, I don't think I have a test for that…